CONTRIBUTING.md

```diff
@@ -1,11 +1,14 @@
 Mística-icons is built to grow, that means that If you need a new icon, you have to create it following the brand guidelines in https://brandfactory.telefonica.com and then [create an issue](https://github.com/Telefonica/mistica-design/issues/new?assignees=&labels=fundamentals%3A+icons%2C+request+%E2%9C%A8&template=icon_request.md&title=) to add this icon to mistica-icon repository.
 
 # Add new icons
+
 1. Check If mistica-icons doesn't cover your need in [this list](https://github.com/Telefonica/mistica-icons/blob/production/README.md)
 2. Read [Brand Factory](https://brandfactory.telefonica.com/hub/134) icons guidelines by brand
 3. Create a new icon following the guidelines and **with 3 weights!** _(light / regular / filled)_
-4. Create an issue [here](https://github.com/Telefonica/mistica-design/issues/new?assignees=&labels=fundamentals%3A+icons%2Crequest+%E2%9C%A8&template=icon_request.yml&title=Title)
-5. That's all! We will review If your icon have all the needs to add it to mistica-icons
+4. Provide 3 synonyms for this icon. [Keywords file](icons/icons-keywords.json)
+5. Create an issue [here](https://github.com/Telefonica/mistica-design/issues/new?assignees=&labels=fundamentals%3A+icons%2Crequest+%E2%9C%A8&template=icon_request.yml&title=Title)
```

Comment:
Dentro de la propia issue metería un apartado que sea "Provide 3 synonyms for this icon"

Reply:
uuu nice. Lo añado.

Molaría que al añadir un nuevo icono fuese todo automatico, que se añadiera solo al json de keywords y con IA incluyese los sinonimos en todos los idiomas ✨

Reply:
Eso sería lo más!

Reply:
Eso sería lo más!

Reply:
Añadido a la template de las issues para icon request 
https://github.com/Telefonica/mistica-design/pull/1459